bitnami/node-exporter/README.md

```diff
@@ -0,0 +1,141 @@
+# Node Exporter
+
+[Node Exporter](https://github.com/prometheus/node_exporter) is a Prometheus exporter for hardware and OS metrics exposed by *NIX kernels, written in Go with pluggable metric collectors.
```

Comment:
Is the asterisk intended at `*NIX kernels`?

Reply:
yep is expected. The asterisk signifies unix, linux, etc.